src/components/UserInformation.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+<div id="hello-coffee">
+    <p> User Information : </p>
+    <p> - Account: <input ref="txtAccount" class="input" :class="{active: isActive}" :value="user.account"/></p>
+    <p> - Address: <input ref="txtAddress" class="input" :class="{active: isActive}" :value="user.address"/></p>
+    <button @click="edit">{{buttonName}}</button>
+    <router-link :to="{name: 'home'}"><button>Back</button></router-link>
+</div>
+</template>
+
+<script lang="coffee" type="text/coffeescript">
+{ mapGetters} = require 'vuex'
+
+module.exports =
+  data: () ->{
+    buttonName: "Edit",
+    isActive: false,
+    id: parseInt(@.$route.params.id),
+  }
+  methods: {
+    edit: () ->
+      if @.buttonName == "Edit"
+        @.isActive = true;
+        @.buttonName = "Save";
+        @.$refs.txtAccount.focus();
+      else
+        newUser = {account: @.$refs.txtAccount.value, address: @.$refs.txtAddress.value}
```

Comment:
The issue is you get the change by DOM element. How about using `v-bind` provided by Vue? Please think about that and update.